llvm/lib/Target/RISCV/RISCVInstrInfoXCheri.td

```diff
@@ -339,6 +341,7 @@ def PseudoCapTAILIndirect : Pseudo<(outs), (ins GPCRTC:$rs1),
 
 let Predicates = [HasCheri] in {
 def CTestSubset : Cheri_rr<0x20, "ctestsubset", GPR, GPCR>;
+def CSetEqualExact : Cheri_rr<0x21, "csetequalexact", GPR, GPCR>;
```

Comment:
CSetEqualExact should, IMO, be treated like CRRL/CRAM, in that its CSEQX short form is the preferred alias and the internal name of the instruction.